pkgs/applications/altcoins/ledger-live-desktop/default.nix

```diff
@@ -0,0 +1,42 @@
+{
+ stdenv,
+ appimage-run,
+ fetchurl,
+ runtimeShell,
+}:
+
+stdenv.mkDerivation rec {
+  github-user = "LedgerHQ";
+  pname = "ledger-live-desktop";
+  version = "1.9.1";
+
+  src = fetchurl {
+    url = "https://github.com/${github-user}/${pname}/releases/download/v${version}/${pname}-${version}-linux-x86_64.AppImage";
+    sha256 = "0b919ilvv3zi17fnzngnnmg28dxqlq0dvj3fdvb50kh3ibrhdcfz";
+  };
+
+  buildInputs = [ appimage-run ];
+
+  unpackPhase = ":";
+  preInstall = ":";
```

Comment:
@alexarice like this?

Reply:
like https://github.com/NixOS/nixpkgs/blob/8f5edef47e2d7d3667148d7ea8122fb4ec6794d1/pkgs/applications/editors/sublime/3/common.nix#L86